tests/test_s3.py

```diff
@@ -774,45 +776,45 @@ def test_cover_image_url(
             'marc',
             'SHORT',
             'Lane',
-            datetime.datetime(2020, 1, 1, 0, 0, 0),
-            'https://marc.s3.amazonaws.com/SHORT/2020-01-01%2000%3A00%3A00/Lane.mrc'
+            datetime_utc(2020, 1, 1, 0, 0, 0),
+            'https://marc.s3.amazonaws.com/SHORT/2020-01-01%2000%3A00%3A00%2B00%3A00/Lane.mrc'
```

Comment:
I wasn't expecting this but I guess it's better than the alternative.

Reply:
I think we're fine; MARC files uploaded prior to this change will have the old URL stored in their Resource objects.

Reply:
Do those Resource objects need to be converted if they are then read at a later point?

Reply:
I don't think so, since we only mirror these resources once. (If we mirrored it again it would be for a different date and the URL would be different anyway.)